.github/workflows/ci.yml

```diff
@@ -12,7 +12,98 @@ env:
   GOTESTSUM_CMD: "gotestsum --format standard-verbose --debug --"
   GOTESTSUM_CMD_RAW: "gotestsum --format standard-verbose --debug --raw-command -- go tool test2json -t"
 
+  LCOW_ARTIFACT_PROJECT: "ContainerPlatform"
+  LCOW_ARTIFACT_FEED: "ContainerPlat-Dev"
+  LCOW_ARTIFACT_NAME: "azurelinux-uvm"
+  LCOW_ARTIFACT_VERSION: "0.0.13"
```

Comment:
Is there a way to just get whatever the latest release package is for this instead of using a set version? I know in ADO, you can use an "*" to indicate that, do you know if that works for github too?

Reply:
Yes I can use "*.*.*" to indicate the latest. Updated the code to reflect that.